src/eko/kernels/as4_evolution_integrals.py

```diff
@@ -0,0 +1,320 @@
+# -*- coding: utf-8 -*-
+"""This file implements the |N3LO| evolution integrals"""
+
+import numba as nb
+import numpy as np
+
+
+@nb.njit(cache=True)
+def roots(b_list):
```

Comment:
help me I'm too lazy right now to think it through :upside_down_face: , but this roots business should be related to Cauchy in some way, right?

Reply:
I think it's just a property of the integrals `x^n/p(x)`, also the lower order integrals can be written this way...

Reply:
yes, and this "property" should be derived some way (I think) from Cauchy's integral theorem - I seem to remember this example (`p(x)/q(x)`) from my "math for physicists" lecture ... when I remember I can always add some more documentation ...